src/main/java/mods/railcraft/common/carts/EntityTunnelBore.java

```diff
@@ -857,15 +857,22 @@ private boolean canMineBlock(BlockPos targetPos, IBlockState existingState) {
     }
 
     protected float getLayerHardness(BlockPos targetPos, BlockRailBase.EnumRailDirection dir) {
-        float hardness = layerAction(targetPos, dir, 0F, this::getBlockHardness, (s, r) -> s + r);
+        float hardness = layerAction(targetPos, dir, 0F, (pos, direction) -> {
+            float f = getBlockHardness(pos, direction);
+            ItemStack head = getStackInSlot(0);
+            if (canHeadHarvestBlock(head, worldObj.getBlockState(pos))) {
+                int i = EnchantmentHelper.getEnchantmentLevel(Enchantments.EFFICIENCY, head);
+                f /= i * i + 1;
```

Comment:
What is this? This is even worse that before!

Level 1: 2x faster
Level 2: 5x faster
Level 3: 10x faster
Level 4: 17x faster
Level 5: 26x faster

How Tools work:
```
strVsBlock = wood 2 || stone 4 || iron 6 || diamond 8
digSpeed = strVsBlock + enchant * enchant + 1
blockDamagePerTick =  digSpeed  / hardness / 30F

tick() {
    blockDamage += blockDamagePerTick 
    if (blockDamage > 1) destroyBlock()
}
```

A Diamond Pick with Level 5 Efficiency is only 12x faster than a Wooden Pick with Level 0.